common4j/src/main/com/microsoft/identity/common/java/telemetry/relay/AbstractTelemetryRelayClient.java

```diff
@@ -39,33 +36,33 @@
 public abstract class AbstractTelemetryRelayClient<T> implements ITelemetryObserver<T> {
 
     private static final String TAG = AbstractTelemetryRelayClient.class.getSimpleName();
-    // Filters to apply on the events captured.
-    // This basically performs an AND operation on the filters to determine whether the event will be relayed.
-    private final List<ITelemetryEventFilter<T>> eventFilters = new ArrayList<>();
+    // Filter to apply on the events captured.
+    private ITelemetryEventFilter<T> mEventFilter = null;
```

Comment:
It seems we've changed the model of the code here now and only going to allow one filter. What was the rationale behind this change. Are there pros and cons to this approach over the previous approach?